README.md

```diff
@@ -71,6 +71,8 @@ bindings for QWidgets APIs.
 
 ## Contributing to CXX-Qt
 
+[![GitHub repo Good Issues for newbies](https://img.shields.io/github/issues/KDAB/cxx-qt/good%20first%20issue?style=flat&logo=github&logoColor=green&label=Good%20First%20issues)](https://github.com/KDAB/cxx-qt/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) [![GitHub Help Wanted issues](https://img.shields.io/github/issues/KDAB/cxx-qt/help%20wanted?style=flat&logo=github&logoColor=b545d1&label=%22Help%20Wanted%22%20issues)](https://github.com/KDAB/cxx-qt/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) [![GitHub Help Wanted PRs](https://img.shields.io/github/issues-pr/KDAB/cxx-qt/help%20wanted?style=flat&logo=github&logoColor=b545d1&label=%22Help%20Wanted%22%20PRs)](https://github.com/KDAB/cxx-qt/pulls?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) [![GitHub repo Issues](https://img.shields.io/github/issues/KDAB/cxx-qt?style=flat&logo=github&logoColor=red&label=Issues)](https://github.com/KDAB/cxx-qt/issues?q=is%3Aopen)
```

Comment:
As we do not use the standard github label names, these shields do not work correctly on our repository.